.github/workflows/ci.yml

```diff
@@ -0,0 +1,47 @@
+name: Miew CI/CD
+
+on: [push, pull_request]
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+
+    - name: Setup Node.js
+      uses: actions/setup-node@v3
+      with:
+        node-version: 18
+        
+    - name: Cache dependencies
+      uses: actions/cache@v3
+      with:
+        path: |
+          node_modules
+          ~/.sonar/cache
+          ~/.yarn
+        key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-yarn-
+
+    - name: Setup Code Climate test-reporter
+      run: |
+        curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > /tmp/cc-test-reporter
+        chmod +x /tmp/cc-test-reporter
+
+    - name: Install dependencies and run CI script
+      run: |
+        yarn install
+        yarn run ci
+
+  deploy:
```

Comment:
I know, we wanted to keep it. I'm still not sure if we should do it. This is effectively untested code... And Dmitry has a related question we cannot answer)) Let's delete it until we are ready to implement it properly.

The following example makes me think that the job will fail: https://github.com/actions/starter-workflows/blob/main/ci/npm-publish.yml#L21

Reply:
note: I didn't check it now, but as I recall, such printing logic: `{{ secrets.NPM_AUTH_TOKEN }}` just returns `***` in GA (instead the actual value), so if I'm right, even in theory this line will be incorrect.